plotly/graph_objs/graph_objs_tools.py

```diff
@@ -37,7 +37,8 @@ def _list_help(object_name, path=(), parent_object_names=()):
     items = graph_reference.ARRAYS[object_name]['items']
     items_classes = [graph_reference.string_to_class_name(item)
                      for item in items]
-    lines = textwrap.wrap(repr(items_classes), width=LINE_SIZE-TAB_SIZE)
+    items_classes.sort()
+    lines = textwrap.wrap(repr(items_classes), width=LINE_SIZE-TAB_SIZE - 1)
```

Comment:
Nit Picky, but would `lines = textwrap.wrap(repr(items_classes), width=LINE_SIZE - TAB_SIZE - 1)`
or `lines = textwrap.wrap(repr(items_classes), width=LINE_SIZE-TAB_SIZE-1)` be better for consistency?